src/plugins/intel_cpu/src/runtime_env.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cache/multi_cache.h"
+#include "config.h"
+#include "extension_mngr.h"
+#include "dnnl_scratch_pad.h"
+#include "weights_cache.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+struct RuntimeEnv {
```

Comment:
general comment: I would expect all pointers inside to be unmutable. Maybe we can put to private section and provide getters.

Reply:
changed accordingly